tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandlerTest.java

```diff
@@ -1165,7 +1155,7 @@ public void testBrokerHandleTopicMetadataRequestAllowAutoTopicCreation(boolean b
             String topicName = "kopBrokerHandleTopicMetadataRequest-" + brokerAllowAutoTopicCreation + "-"
                     + overrideNameSpaceAutoTopicCreation + "-"
                     + allowAutoTopicCreationInRequest;
-            List<String> kafkaTopics = Arrays.asList(topicName);
+            List<String> kafkaTopics = List.of(topicName);
```

Comment:
Please revert this change because this PR could be cherry-picked to older branches that are compatible with Java 8

Reply:
Just found we can use `Collections.singletonList` here because the list can be immutable.

Reply:
+1